person.rb

```diff
@@ -0,0 +1,21 @@
+class Person
+  attr_reader :id
+  attr_accessor :name, :age
+
+  def initialize(age, name = 'Unknown', parent_permission: true)
+    @id = Random.rand(1..1000)
+    @name = name
+    @age = age
+    @parent_permission = parent_permission
+  end
+
+  def of_age?
+    @age >= 18
+  end
+  private :of_age?
+
+  def can_use_services?
+    of_age? || @parent_permission
+  end
+  public :can_use_services?
+end
```

Comment:
Nice job creating the `Nameable` class and implementing the required features 👍. Just as required in this milestone, kindly ensure your `person` class inherits the properties of the `Nameable`👍. Also, ensure the class has a method `correct_name`  that returns the name attribute as required👍. 
<img width="704" alt="image" src="https://github.com/MICHAELKITH/oop-library/assets/64643995/4fb0b72b-743e-44b4-864b-6f22e46e0722">